CnpSdkForNet/CnpSdkForNet/Communications.cs

```diff
@@ -536,15 +553,14 @@ public virtual void FtpPoll(string fileName, int timeout, Dictionary<string, str
             sftpClient.Disconnect();
         }
 
-        public virtual void FtpPickUp(string destinationFilePath, Dictionary<string, string> config, string fileName)
+        public virtual void FtpPickUp(string destinationFilePath, string fileName)
         {
             SftpClient sftpClient;
 
-            var printxml = config["printxml"] == "true";
-
-            var url = config["sftpUrl"];
-            var username = config["sftpUsername"];
-            var password = config["sftpPassword"];
+            var printxml = _config["printxml"] == "true";
```

Comment:
Maybe switch to bool.Parse, so an exception is raised if config is wrong. May be helpful in debugging if someone has a slight typo they didn't realize.